tftpsync/susemanager-tftpsync-recv/configure-tftpsync.sh

```diff
@@ -163,8 +163,8 @@ else
 fi
 
 
-sed -i "s/^[[:space:]]*Allow from[[:space:]].*$/        Allow from $SUMA_IP/" /etc/apache2/conf.d/susemanager-tftpsync-recv.conf
-sed -i "s/^[[:space:]#]*Require ip[[:space:]].*$/        Require ip $SUMA_IP/" /etc/apache2/conf.d/susemanager-tftpsync-recv.conf
+sed -i "s/^[[:space:]]*Allow from[[:space:]].*$/        Allow from $SUMA_FQDN/" /etc/apache2/conf.d/susemanager-tftpsync-recv.conf
+sed -i "s/^[[:space:]#]*Require ip[[:space:]].*$/        Require ip $SUMA_FQDN/" /etc/apache2/conf.d/susemanager-tftpsync-recv.conf
```

Comment:
```suggestion
sed -i "s/^[[:space:]#]*Require (ip|host)[[:space:]].*$/        Require host $SUMA_FQDN/" /etc/apache2/conf.d/susemanager-tftpsync-recv.conf
```

Reply:
I have reverted the host check. So this comment is not needed anymore.